src/components/ProductListContainer/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ProductCard from '../ProductCard';
+import './styles.scss';
+
+const fakeProductsList = [
+    {
+        id: 1,
+        productPhotoUrl: 'https://placehold.co/310x210?text=Imagen',
+        listPrice: '$250.000,00',
+        discountedPrice: '$200.000,00',
+        discountPercentage: 20,
```

Comment:
It isn't included as prop of ProductCard but I added it 'cause I think is necessary for display it at the DiscountBadge later.